### PR TITLE
CF Actor - Add enable_termination_protection option

### PR DESCRIPTION
**Pull request by nehalrp**
@diranged Still need to finish updating docs and tests, but I wanted to get your eyes on the implementation here.

### CODE REVIEW

**Comment by stlava** [comment]
@nehalrp do you know if this also puts termination protection on resources under the stack as well or to the stack itself?

**Comment by nehalrp** [comment]
@stlava It's purely on the stack itself.

**Comment by nehalrp** [comment]
@diranged Updated with tests.

File: kingpin/actors/aws/cloudformation.py [hunk 1]
```diff
@@ -762,6 +781,9 @@ class Stack(CloudFormationBaseActor):
         'timeout_in_minutes': (int, 60,
                                'The amount of time that can pass before the '
                                'stack status becomes CREATE_FAILED'),
+        'enable_termination_protection': (TerminationProtectionConfig, None,
```
Comment (diranged) [inline kingpin/actors/aws/cloudformation.py]: Add to the docstrings above fro this option - also while you're there, can you include the `role_arn` docs too?

Comment (nehalrp) [reply kingpin/actors/aws/cloudformation.py]: Ah, missed those particular ones.

**Pull request merged**